src/github.com/matrix-org/dendrite/roomserver/storage/rooms_table.go

```diff
@@ -11,7 +12,11 @@ CREATE TABLE IF NOT EXISTS rooms (
     -- Local numeric ID for the room.
     room_nid BIGINT PRIMARY KEY DEFAULT nextval('room_nid_seq'),
     -- Textual ID for the room.
-    room_id TEXT NOT NULL CONSTRAINT room_id_unique UNIQUE
+    room_id TEXT NOT NULL CONSTRAINT room_id_unique UNIQUE,
+	-- The most recent events in the room that aren't referenced by another event.
```

Comment:
Naming: Whilst I appreciate you've been consistently calling this "latest" events, I don't think it's a good name. I think of latest as:
```
Time goes down the page. 1 is the m.room.create event (root).

    1
    |
    2
   /  \
  3   4
  |   |
  5   6 <--- latest
  |
  7 <----- latest
```

whereas in actuality, it seems "latest" is anything which doesn't match up to the known state of the DAG, i.e. no known parents (orphans?):

```
    1
    |
    2
   /  \
  3   4
  |   |
  5   6
  |
  7   8 <--- latest
```

Is my thinking correct? If so, why not call these `orphan_events`?